status/versions.go

```diff
@@ -308,22 +315,83 @@ func kubernetesVersion() (*ExternalServiceInfo, error) {
 	return nil, err
 }
 
-// set this one time, it is very unlikely that the istio version will change without a kiali pod restart, or if it
-// did that that version change will matter, and the kiali pod could be bounced as a workaround.
-var istioSupportsCanonical *bool
+// set this one time, it is very unlikely that mixer will be enabled/disabled without a kiali pod restart, or if it
+// did that that change will matter, and the kiali pod could be bounced as a workaround.
+var isMixerDisabled *bool
+
+// IsMixerDisabled returns true if Telemetry V2 is enabled (mixer is not collecting metrics)
+// TODO: This test can be removed when Kiali stops supporting Istio versions with Mixer Telemetry
+func IsMixerDisabled() bool {
+	if isMixerDisabled != nil {
+		return *isMixerDisabled
+	}
+
+	k8sConfig, err := kubernetes.ConfigClient()
+	if err != nil {
+		log.Warningf("IsMixerDisabled: Cannot create config structure Kubernetes Client.")
+		return false
```

Comment:
I think this should probably return true.  Going forward more users will have Telem V2, I think.  Honestly there isn't really a good default, if we can't determine this it's a pretty big problem.

Reply:
I was keeping false as default, just because it is the "default" in Istio side at least when speaking about how that flag is defined their code. But the real "default" when installing Istio 1.5 is "true" and Kiali is focusing support on the latest Istio version. So, I agree this should be `true`.